core/dbt/deps/base.py

```diff
@@ -97,6 +99,34 @@ def get_installation_path(self, project, renderer):
     def get_subdirectory(self):
         return None
 
+    def _install(self, project, renderer):
+        metadata = self.fetch_metadata(project, renderer)
+
+        tar_name = "{}.{}.tar.gz".format(self.package, self.version)
```

Comment:
Minor detail:  We prefer using the more modern [f-string style python string formatting](https://peps.python.org/pep-0498/).

Reply:
Thanks Ian - I've changed it over to f-string with 8787ba41af585b954d18c24c2dccbf147035cd9f